Update account signing strings for v1.2 release

m52go opened the pull request:
I'm making a couple of in-line comments -- please see them.

EDIT

Cannot comment on unchanged code...I'll just add comments here.

---

```
offerbook.timeSinceSigning.info.peerLimitLifted=signed by a peer and limits were lifted
offerbook.timeSinceSigning.info.signer=signed by peer and can sign peer accounts
```

Same thing now?

---

Changed `popup.warning.tradeLimitDueAccountAgeRestriction` to fit new rules (since March 1 2019 date is no longer the determining factor for signing).

---

Changed single quotes to double quotes in `portfolio.pending.mediationResult.popup.info`...seems like mistake since variable includes {n} insertions.

Commit:
Update account signing strings for v1.2 release

ripcurlx commented:
> Same thing now?

The `offerbook.timeSinceSigning.info.peerLimitLifted` won't be displayed right now, but I'd like to keep it there if we decide to change this again later. Still I think we should extend the `offerbook.timeSinceSigning.info.signer` value with the limits lifted state. What do you think?

ripcurlx commented:
e.g. `offerbook.timeSinceSigning.info.signer=signed by peer and can sign peer accounts (limits lifted)`?

ripcurlx commented:
Or we leave this info out, as it might not be of any help for the other trading peer?

ripcurlx commented on core/src/main/resources/i18n/displayStrings.properties:
```diff
@@ -376,15 +376,17 @@ offerbook.warning.noMatchingAccount.msg=To take this offer, you will need to set
 
 popup.warning.tradeLimitDueAccountAgeRestriction.seller=The allowed trade amount is limited to {0} because of security restrictions based on the following criteria:\n\
- - The buyers account was created after March 1st 2019\n\
+ - The buyer''s account is not at least 60 days old
```
You are missing a `\n\` at the end of the line

ripcurlx commented:
> Changed `popup.warning.tradeLimitDueAccountAgeRestriction` to fit new rules (since March 1 2019 date is no longer the determining factor for signing).

I think this change is not correct. The criteria should be something like this:
- The buyer''s account has not been signed by an arbitrator or a peer
- The time since signing of the buyer''s account is not at least 30 days 
- The payment method for this offer is considered risky for bank chargebacks

Commit:
Add minor corrections from ripcurlx review

Commit:
Adjust tradeLimitDueAccountAgeRestriction string

So that it describes why an account isn't signed (in general) instead of
why it wasn't signed by an arbitrator.

m52go commented:
> Or we leave this info out, as it might not be of any help for the other trading peer?

I added the change -- I think it's better to be as clear as possible.

> I think this change is not correct. The criteria should be something like this:

Makes sense, changed. I was under the impression that this string describes why payment accounts were not signed by an arbitrator at first, not why they aren't signed in general.

ripcurlx commented:
> Makes sense, changed. I was under the impression that this string describes why payment accounts were not signed by an arbitrator at first, not why they aren't signed in general.

It is triggered whenever the seller's or buyer's account has the 0.01 BTC limit. In the past that was for all accounts after 1st of March 2019. Good that you found that this translation needed to be changed as well 👍

ripcurlx reviewed (approved):
utACK
